exp/api/v1alpha3/azuremanagedcontrolplane_types.go

```diff
@@ -81,6 +81,10 @@ type AzureManagedControlPlaneSpec struct {
 	// AadProfile is Azure Active Directory configuration to integrate with AKS for aad authentication.
 	// +optional
 	AADProfile *AADProfile `json:"aadProfile,omitempty"`
+
+	// Sku is the SKU of the AKS to be provisioned.
+	// +optional
+	Sku *SKU `json:"sku,omitempty"`
```

Comment:
minor nit: capitalization inconsistency

Reply:
should we be adding this to v1a3 at this point? or just doing webhook conversion?

Reply:
cc @CecileRobertMichon 

i'd assume the latter

Reply:
@alexeldeib thanks for reviewing my PR!

capitalization inconsistency: i'll address it :)

v1a3 vs. v1a4: I just tried reverting the change to v1a3, but the hub-spoke-hub test of the v1a3 [conversion test](https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/exp/api/v1alpha3/conversion_test.go) would fail since it's lossy (we don't store sku in v1a3). So my understanding is that we still need to add it to v1a3. Is that correct?

Reply:
This is not correct, roundtrip conversion should not be lossy. See how it’s handled for the identity ref which is not present in v1a3.

Stored the higher version and unmarshal it inside the conversion webhook, then rehydrate the field from the unmarshaled struct.

Reply:
ah interesting, I reverted change in v1a3 and added one line in `azuremanagedcontrolplane_conversion.go` to restore the field, now it works. Thanks @alexeldeib !